clang/include/clang/Driver/Options.td

```diff
@@ -6060,29 +6060,36 @@ def fsycl_is_host : Flag<["-"], "fsycl-is-host">,
 // CAS Options
 //===----------------------------------------------------------------------===//
 
-// Example command-lines we expect (not all supported yet):
+// Example command-lines we expect:
+//
+// For building against some pre-ingested, CAS-based filesystem in the builtin
+// on-disk CAS:
+// ```
+// % clang -fcas=on-disk -fcas-fs=llvmcas://1af12fa2afa1af2a1fa
```

Comment:
`-fcas=on-disk` has reappeared in the comment, probably a mistake?

Reply:
fixed in b349b5b897294705b65da1c68e904f95ae509b8d